packages/gatsby/src/commands/serve.ts

```diff
@@ -133,6 +134,17 @@ module.exports = async (program: IServeProgram): Promise<void> => {
       `/api/*`,
       multer().none(),
       express.urlencoded({ extended: true }),
+      (req, res, next) => {
```

Comment:
https://www.npmjs.com/package/cookie-parser is the standard parser but I doubt we'll want to directly support JSON cookies (never used those personally) or signed cookies (people can handle this themselves on per-route basis)